### PR TITLE
Removal of the Application Registry, Connector Service and Connection Token Handler pipelines

### PR DESCRIPTION
**Pull request by mvshao**
**Description**
We are planning to remove the Application Registry, Connector Service and Connection Token Handler components, so in this case, we won't need these pipelines.

Changes proposed in this pull request:

- remove the pipelines responsible for Application Registry, Connector Service and Connection Token Handler

**Related issue(s)**
See also https://github.com/kyma-project/kyma/issues/13718

### CODE REVIEW

**Review by koala7659 — COMMENTED** [review]
Please check my comments

File: templates/data/generic_component_data.yaml [hunk 1]
```diff
@@ -25,30 +25,6 @@ templates:
                     global:
                       - "jobConfig_postsubmit"
                       - "disable_testgrid"
-      - to: ../prow/jobs/kyma/components/application-registry/application-registry-generic.yaml
```
Comment (koala7659) [inline templates/data/generic_component_data.yaml:28]: application-connectivity-certs-setup-job configuration (above) should also be removed

**Review by koala7659 — APPROVED** [review]
LGTM

**Review by koala7659 — APPROVED** [review]
LGTM

**Review by a-thaler — DISMISSED** [review]

**Review by Ressetkk — APPROVED** [review]
I approve removing more stuff :)
/joke

**Review by grego952 — APPROVED** [review]

**Pull request merged**